src/mmda/types/span.py

```diff
@@ -53,3 +53,14 @@ def small_spans_to_big_span(cls, spans: List["Span"]) -> "Span":
             end=end,
             box=Box.small_boxes_to_big_box(boxes=[span.box for span in spans]),
         )
+
+    def is_overlap(self, other: "Span") -> bool:
+        return self.start < other.end and self.end > other.start
```

Comment:
what about if `self.start > other.end and self.end > other.start` (other encompasses self)? would it make sense to put `is_overlap_1d` ([this](https://github.com/allenai/mmda/blob/58a2bfe40c8078525bc616c2665e9d5a9ae3834d/src/mmda/types/box.py#L14)) in a more common place and use it here same as `box`? Is there something different about trying to check overlap that way when dealing with Spans?